frontend/src/pages/RenterCandidatePage.tsx

```diff
@@ -158,66 +345,298 @@ export function RenterCandidatePage() {
                 alt={"Back arrow"}
                 style={{ marginRight: "12px" }}
               />
-              Back
+              Back to Listing
```

Comment:
Users will be able to access the Renter Candidate page from the All Renter Candidate page (once it's implemented) as well. Thoughts on calling this "Back" to keep it more general?
```suggestion
              Back 
```